lua/code-link/clipboard.lua

```diff
@@ -2,7 +2,7 @@ local static = require("code-link.static")
 local ClipBoard = {}
 
 function ClipBoard.copy_or(content)
-	if static.config.copy_command == nil then
+	if static.config == nil or static.config.copy_command == nil then
```

Comment:
Seems like maybe I can define a default config before using it to avoid determining whether the config is nil

Reply:
Recently I've written a plugin template that contains some useful scripts and the common directory structure of Neovim lua plugins, I was wondering if it would be better to put the config-parse logic to the template, and the code-link plugin should apply this template

Reply:
Using a default config is probably preferred - at least that's what I've seen many other plugins do. :slightly_smiling_face: